scripts/generate-tokens-wrapper.js

```diff
@@ -134,13 +134,77 @@ const processTokens = (srcPath, tokensDir) => {
         return;
     }
 
-    fs.writeFileSync(
-        path.join(__dirname, '..', 'tools', 'styles', tokensDir, fileName),
-        css
-    );
+    try {
+        fs.writeFileSync(
+            path.join(__dirname, '..', 'tools', 'styles', tokensDir, fileName),
+            css
+        );
+    } catch (er) {}
 };
 
 const processPackages = async (srcPath, tokensDir, index) => {
+    if (tokensDir === 'tokens') {
+        const packagename = tokenPackages[index];
+
+        let componentLevelTokensPath = path.join(
+            __dirname,
+            '..',
+            'node_modules',
+            '@spectrum-css',
+            tokensDir,
+            'dist',
+            'css',
+            'components'
+        );
+        const spectrumPath = path.join(
+            componentLevelTokensPath,
+            'spectrum',
+            packagename + '.css'
+        );
+        // check if spectrumPath exists
+        if (fs.existsSync(spectrumPath)) {
+            let spectrum = fs.readFileSync(spectrumPath, 'utf8');
+            spectrum = removeImporantComments(targetHost(spectrum));
```

Comment:
```suggestion
            spectrum = removeImportantComments(targetHost(spectrum));
```

Small spelling note ^

Reply:
Oh question - I'm assuming this function removes the copyright which you're also adding to files - why not just keep the copyright and strip duplicates after merge?  The `postcss-discard-comments` plugin offers a `removeAllButFirst` option which is a nice way to preserve the first copyright in the file.